app/controllers/users_controller.rb

```diff
@@ -83,6 +84,9 @@ def login
         login_user(user)
       end
     else
+      if @brute_force_attempt
+        inline_error _("Too many tries, please try again in a few minutes.")
```

Comment:
Can you be more specific saying "five minutes" or is this on purspose? :-)

Reply:
on purpose :) not that it would be hard to figure out, but no point saying when the attacker should resume.